deploy.md

```
@@ -0,0 +1,118 @@
+Médio (operacionalização)
+3 horas
+Todos os itens acima
+Essa priorização segue o princípio de MVP (Minimum Viable Product) para deploy, focando em estabilidade antes de otimizações avançadas como CI pipelines.
+3. Plano de Ação Detalhado
+Implemente os itens em sequência, validando localmente com docker compose up -d antes do push para registry. Assuma um registry privado na VPS (ex.: registry.sua-vps.com).
+Passo 1: Refatorar o Dockerfile (backend/Dockerfile)
+	•	Adote estrutura multi-stage: # Stage 1: Build
+	•	FROM python:3.11-slim AS builder
+	•	WORKDIR /app
+	•	COPY requirements.txt .
+	•	RUN pip install --no-cache-dir -r requirements.txt && pip install Swiss Ephemeris
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)